scimono-server/src/main/java/com/sap/scimono/callback/resourcetype/DefaultResourceTypesCallback.java

```diff
@@ -16,4 +18,9 @@
   public ResourceType getCustomResourceType(String typeId) {
     return null;
   }
+
+  @Override
+  public List<SchemaExtension> getSchemaExtensions(String resourceTypeId) {
```

Comment:
You don't need to override this method anymore.

Reply:
Removed.